src/manifold/CMakeLists.txt

```diff
@@ -26,3 +26,15 @@ target_link_libraries(${PROJECT_NAME}
 target_compile_features(${PROJECT_NAME}
     PUBLIC cxx_std_17
 )
+
+install(
+  TARGETS ${PROJECT_NAME}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR}
+  COMPONENT runtime
+)
+install(
+  DIRECTORY include/
+  DESTINATION ${CMAKE_INSTALL_FULL_INCLUDEDIR}/manifold
+  COMPONENT devel
```

Comment:
I love it! Are we testing the install somehow in our CI?

Reply:
the nix build uses install, but we are not checking the file contents